### DISCUSSION TITLE
想问下该怎么开机自动启动呀

### DISCUSSION THREAD

**Discussion by chenchen7427** [aliencaocao/netease_cloudmusic_discord_rpc#32] [Ideas]
想问下该怎么开机自动启动呀

**Comment by aliencaocao** [comment]
目前不支持这个功能，以后会加

**Comment by aliencaocao** [comment]
0.3.4版本已添加此功能：https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/releases/tag/0.3.4
